docs/ecm-user.model.md

```diff
@@ -0,0 +1,43 @@
+# Ecm User model
+
+Contains information about a Content Services user.
+
+## Details
+
+Instances of this class are returned by the methods of the
+[Ecm User service](ecm-user.service.md). It implements the `Person`
+interface, which is defined in the
+[Alfresco JS API](https://github.com/Alfresco/alfresco-js-api/blob/master/src/alfresco-core-rest-api/docs/Person.md).
+
+```ts
+class EcmUserModel implements Person {
```

Comment:
I prefer to not have a copy of the model in the doc could be easily go out of sync